### DISCUSSION TITLE
Testing different websites and now I can't go Public

### DISCUSSION THREAD

**Discussion by normanddeon** [community/community#37551] [New to GitHub]
I tried a few websites by downloading the Zip, imporint in GitHub Desktop  and pushing to GITHUB.  Suddenly today, it won't let me go frim Private to Public unless I upgrade.  Thoughts?

**Comment by ossanna16** [comment]
Hi there @normanddeon and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
it's even in the past, you cannot publish on GitHub Pages from a private repo unless you upgrade,

maybe you just tried that first public then private then you can't publish it again
